pkg/nexus/dependency.go

```diff
@@ -0,0 +1,40 @@
+package nexus
+
+import (
+	"errors"
+	"fmt"
+	"os"
+	"path/filepath"
+)
+
+func ExtractDependecyMetadata(buildFolder string) ([]Dependency, error) {
+
+	metadata := []Dependency{}
+	fileList := []string{}
+	err := filepath.Walk(buildFolder, func(path string, f os.FileInfo, err error) error {
+		fileList = append(fileList, path)
+		return nil
+	})
+
+	if err != nil {
+		return nil, errors.New(fmt.Sprintf("Failed when traversing folder %s", buildFolder))
+	}
+
+	for _, file := range fileList {
+		info, err := os.Stat(file)
+		if err == nil && info.IsDir() {
+			continue
+		}
+		sha1, _ := hashFileSHA1(file)
```

Comment:
SHA1 is broken.

Det vil bety veldig lite her, men er det noen gode grunner for å bruke sha1 og ikke sha2 eller sha3?

Reply:
Nexus bruker fortsatt sha1. Tenkte det var enklere om vi kunne bruke den samme shaen mot nexus. F.eks mot nexusiq